artifactory/utils/npm/config-set.go

```diff
@@ -0,0 +1,24 @@
+package npm
+
+import (
+	gofrogcmd "github.com/jfrog/gofrog/io"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+)
+
+func ConfigSet(key, value, executablePath string) error {
```

Comment:
Let's add a description for this function, to male it easier for the random reader landing on this file.